protos/sources/vega/vega.proto

```diff
@@ -776,6 +776,8 @@ enum TransferType {
   TRANSFER_TYPE_CLEAR_ACCOUNT = 24;
   // Balances restored after network restart
   TRANSFER_TYPE_CHECKPOINT_BALANCE_RESTORE = 25;
+  // Insurance pool fraction transfer from parent to successor market.
```

Comment:
```suggestion
  // Transfer of insurance pool fraction from parent to successor market.
```

Reply:
Unresolving because it looks like we have a second `transfer` still there:
```
// Transfer of insurance pool fraction transfer from parent to successor market.
```